### ISSUE TITLE
Separate Pages from Components

### ISSUE THREAD

**Issue by paul-rinaldi** [codeformilwaukee/DECARCERATION-PLATFORM#81]
**Is your feature request related to a problem? Please describe.**
Currently, there are pages and components in the same folder called components, it's not easy to navigate by a pogrammer.

**Describe the solution you'd like**
/src/components/ (components folder)
/src/pages/ (pages folders)

**Comment by hdenisenko** [comment]
Quick question about this - does this mean for something like `/src/components/CalendarPage` you want it split into `/src/page/CalendarPage/index.js` and the modal components in the components folder?

**Comment by Caryyon** [comment]
it sounds like you want
/pages to be a place for actual routing && /components to be components that can be used within those pages.
this can help with confusion of where is the page implemented and where is what it's built out of.
is this going to mess things up for everyone else's branch at this point?

**Comment by hdenisenko** [comment]
that's a good question - there'll probably be a fair amount of merge conflicts. @Caryyon @paul-rinaldi is this a change we want to introduce while others are working on other branches?

**Comment by paul-rinaldi** [comment]
Yes @Caryyon has it right, this should be done later, I'm making more relevant issues soon (tech debt like this isn't too bad at all)
